wkz/file_helper/fit_collector.py

```diff
@@ -105,8 +118,27 @@ def try_to_mount_device():
         else:
             log.warning("could not mount device")
     else:
-        log.warning("Found Garmin device, but could not mount it.")
+        log.warning(f"Found Garmin device, but could not mount it. {mount_output}")
 
 
 def _mount_device_using_gio(bus: str, dev: str) -> str:
     return subprocess.check_output(["gio", "mount", "-d", f"/dev/bus/usb/{bus}/{dev}"]).decode("utf-8")
+
+def _mount_device_using_pmount(dev: str) -> str:
+    return subprocess.check_output(["pmount", dev, "garmin"]).decode("utf-8")
```

Comment:
Couldn't we do something like:
```suggestion
    subprocess.check_output(["pmount", dev, "garmin"]).decode("utf-8")
    return "Mounted at /media/garmin"
```
and thus avoid the above:
```
...
else:
    mount_output = "Mounted at /media/garmin"
    return mount_output
```
?

Reply:
Yes, That would also work, the reason I choose for the current solution was that I thought the `subprocess.check_output` was used for the try and except blocks. 

If we remove those blocks as suggested above, I think this is a much cleaner solution.

Reply:
If `subprocess.check_output()` should fail and raise an exception, the exception would be propagate to the try and except block in `wkz.api.mount_device_endpoint` even with the above suggested change.